app/helpers/application_helper/button/vm_reset.rb

```diff
@@ -0,0 +1,13 @@
+class ApplicationHelper::Button::VmReset < ApplicationHelper::Button::Basic
+  needs_record
+
+  def skip?
+    !@record.is_available?(:reset)
+  end
+
+  def disable?
+    if @record.is_available_now_error_message(:reset)
+      return @record.is_available_now_error_message(:reset)
```

Comment:
@durandom AFAIK, it's the title on the button that displays on hover, if the button is disabled.